### DISCUSSION TITLE
Shortcut

### DISCUSSION THREAD

**Discussion by bricebis** [emoacht/Monitorian#335] [Q&A]
Hi,
Does Monitorian and its add-on features accept a shortcut by pressing a single key (e.g. F1, not a key combination), like on the "Logitech MX keys" keyboard ?

Thx,
iala

**Comment by emoacht** [comment]
I guess you are talking about hot keys.
The answer is no. It is not valid hot key and impossible.

**Comment by bricebis** [comment]
Thx, Have a good day.

**Comment by bricebis** [comment]
Hi,
I found a solution. I tried your add-on because the Logitech Mx Keybord offers the oppotunity to link a hot key to a shortcut. And it works ! So i can press the hotkeys to modify the brightness of my external screens. Nice !

You can sell your propostion to Microsoft Windows i think.

Thank you. I will leave a comment on the french Microsoft Store... I think also you could informate your people.

Ciao.
Brice

**Reply by emoacht** [reply]
It's good. Yes, some keyboards have the function to send hot keys (key combinations) by hitting one key. I have done the same with Elgato Stream Deck before.

**Reply by jeannotdamoiseaux** [reply]
How did you make this work? I'm using a Keychron K3 with similar brightness buttons. However, the brightness is stuck at 0. I could remap the key's functionality, but I don't know to which button or action to make it adjust the brightness in Monitorian.

**Reply by jeannotdamoiseaux** [reply]
Just now I see it might be an add-on functionality behind a pay wall. I would be willing to pay for this functionality, but not in a subscription form.

**Comment by Ander3232** [comment]
What?? I tried this because the screenshot on Softpedia showed hot-key settings—but I couldn't find them in the program. Now I'm reading that to use a hot key with this, you have to _subscribe_ to it? Just to use a hot key? Really? People didn't used to be so money-hungry. 🤷🏽‍♂️

**Comment by eriksten** [comment]
How can I subscribe to the monitorian addons?

**Comment by Ennazk** [comment]
Use Twinkle Tray. Free, open source, comes with hotkeys.
